apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/call-to-action-02.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Call to Action pattern.
+ *
+ * @package A8C\FSE
+ */
+
+$markup = '
+<!-- wp:cover {"url":"https://dotcompatterns.files.wordpress.com/2020/04/louis-hansel-shotsoflouis-d0ueoxdvj5i-unsplash.jpg","id":181,"dimRatio":20,"customOverlayColor":"#000000","focalPoint":{"x":0.5,"y":"0.70"},"minHeight":620,"align":"full"} -->
```

Comment:
The value for `url` should also be `#1$s` to make sure the attribute and content match and the block doesn't break . It's quite a bit of an edge case since the images we use are not locale-specific and probably won't ever be translated. Alternatively you could also not separate out the image url at all.

Reply:
Oh. That was an oversight. I'll fix it. :)